RecoEgamma/EgammaElectronAlgos/interface/ElectronSeedGenerator.h

```diff
@@ -67,13 +67,12 @@ class ElectronSeedGenerator {
   void seedsFromThisCluster(edm::Ref<reco::SuperClusterCollection> seedCluster,
                             float hoe1,
                             float hoe2,
+                            reco::BeamSpot const& beamSpot,
+                            std::vector<reco::Vertex> const* vertices_,
```

Comment:
please drop the trailing underscore to not confuse with a data member

Reply:
sure!